package.json

```diff
@@ -627,17 +618,17 @@
             },
             "default": {}
           },
-          "dataEditor.omegaEditPort": {
+          "dataEditorOmegaEditPort": {
```

Comment:
Should this just be `omegaEditPort`?

Reply:
I believe they should all keep `dataEditor` so we are aware of what they are for. These ones are used for getting default values.